hello-core-logic/createBoardMimicTrello.js

```diff
@@ -0,0 +1,34 @@
+// Write a function named createBoard that takes one argument, boardName, then creates a board if there is no board with
+// the same name (meaning that it adds a board to the object assigned to the variable, hello.) The function should return,
+// Board '<board-name>' was created. If there is already a board with the same name, it should return Board already exists.
+// Use your listBoards function to display before and after creating a board to test that it worked successfully.
+
+const hello = {
+    'Tester Board': {
+      'To Do': ['Laundry', 'Buy Apples', 'Pay Phone Bill'],
+      'Doing': ['Laundry', 'Studying Javascript', 'Studying HTML', 'Studying Ruby'],
+      'Done': ['Laundry']
+    },
+    'Dreams': {
+      'Wish List': ['Conquer the Seven Kingdoms', 'Get my baby back', 'My hand needs to chill']
+    }
+};
+
+board = "Cuisini";
+
+function createBoard(obj, boardName) {
+  let arr = Object.keys(obj);
+  arr.forEach(function(key) {
```

Comment:
Be careful using the `forEach` function. When you `return` from within a `forEach`, it does NOT stop execution. So the loop will continue.
Also, this code does not function the way you think it does.

When I ran the following:
```javascript
createBoard(hello,'Tester Board');
```
I got `"Board 'Tester Board' was created!"` as the result.

What actually happened here was you have now set `hello['Tester Board'] = []`, so our previous tester board is now destroyed!